components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -1109,12 +1220,45 @@ void RRTMGPRadiation::run_impl (const double dt) {
   }
 
 }
+
+// =========================================================================================
+// This is a copy of a the helper field functionality from nduging...
+// We should abstract this out of these places to somewhere more shared...
+// TODO: Ask Luca what he prefers to do (somewhere in share/util maybe?)
+Field RRTMGPRadiation::create_helper_field(const std::string &name,
```

Comment:
We do have a way to create "internal" fields, see: https://github.com/E3SM-Project/scream/blob/master/components/eamxx/src/share/atm_process/atmosphere_process.hpp#L192, however there are other processes where an internal field is not appropriate and they too have a create_helper_field defined.  I think this task might be to consolidate internal and helper fields, if that is possible.

Reply:
@bartgol ^^. Maybe you have some insight on the use of internal fields.